src/super_gradients/training/pipelines/pipelines.py

```diff
@@ -196,7 +199,7 @@ def _generate_prediction_result_single_batch(self, images: Iterable[np.ndarray])
             postprocessed_predictions.append(prediction)
 
         # Yield results one by one
-        for image, prediction in zip(images, postprocessed_predictions):
+        for image, prediction, preprocessed_image in zip(images, postprocessed_predictions, preprocessed_images):
```

Comment:
That's a leftover right ? Because I dont see `preprocessed_image` being used

Reply:
correct, fixed (1a3439dc1900e610f268a2dc75044fe2f1cb0be3)